compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/ConstantValueResolver.java

```diff
@@ -579,6 +581,10 @@ BLangConstantValue constructBLangConstantValueWithExactType(BLangExpression expr
 
         this.anonTypeNameSuffixes = anonTypeNameSuffixes;
         updateConstantType(constantSymbol, expression, env);
+        BLangTypeDefinition typeDefinition = createdTypeDefinitions.get(constantSymbol.type.tsymbol);
+        if (isSourceOnlyAnon && typeDefinition != null) {
```

Comment:
If we do
```suggestion
        BLangTypeDefinition typeDefinition = isSourceOnlyAnon ? createdTypeDefinitions.get(constantSymbol.type.tsymbol) : null;
        if (typeDefinition != null) {
```
we can avoid unnecessary `get` to the map but might affect readability. WDY think?

Reply:
+1